### ISSUE TITLE
No metrics found. Looks like you didn't install node-agent. after helm install

### ISSUE THREAD

**Issue by emoxam** [coroot/coroot#16]
Because i need prometheus with AlertManager (for robusta) - i am using Kube-Prometheus-Stack
i used 
helm install --namespace coroot --create-namespace coroot --set prometheus.enabled=false coroot/coroot
And now coroot shows me error "No metrics found. Looks like you didn't install node-agent."
But agents is there and it's working, what did i miss ?

k get pod -A | grep agent
```
coroot          coroot-node-agent-lcd97                                  1/1     Running     0              26m
coroot          coroot-node-agent-zjqjv                                  1/1     Running     0              26m
coroot          coroot-node-agent-zr9z7                                  1/1     Running     0              26m
```

![image](https://user-images.githubusercontent.com/3593601/212096544-07dbeac8-911b-4339-9f8d-bacaa1df2d8d.png)

**Comment by def** [comment]
Make sure that your Prometheus has discovered node-agent's endpoints. If you use Prometheus Operator, create a PodMonitor as described [here](https://coroot.com/docs/metric-exporters/node-agent/installation).

**Comment by emoxam** [comment]
`k describe PodMonitor -n coroot
Name:         coroot-node-agent
Namespace:    coroot
Labels:       <none>
Annotations:  <none>
API Version:  monitoring.coreos.com/v1
Kind:         PodMonitor
Metadata:
  Creation Timestamp:  2023-01-12T14:46:33Z
  Generation:          1
  Managed Fields:
    API Version:  monitoring.coreos.com/v1
    Fields Type:  FieldsV1
    fieldsV1:
      f:metadata:
        f:annotations:
          .:
          f:kubectl.kubernetes.io/last-applied-configuration:
      f:spec:
        .:
        f:podMetricsEndpoints:
        f:selector:
    Manager:         kubectl-client-side-apply
    Operation:       Update
    Time:            2023-01-12T14:46:33Z
  Resource Version:  22384541
  UID:               47b99f6b-7533-4abf-84b9-5af55c8a5b17
Spec:
  Pod Metrics Endpoints:
    Port:  http
  Selector:
    Match Labels:
      App:  coroot-node-agent
Events:     <none>
`
I created.  And in 'k edit prometheus' i changed

```
  podMonitorSelector:
    matchLabels:
      release: robusta
```
to
```
 podMonitorNamespaceSelector: {}
  podMonitorSelector: {}
```

**Comment by def** [comment]
It seems that in the helm chart we did not set the `app` label for `node-agent`. As a workaround, `app.kubernetes.io/name=node-agent` can be used:
```
apiVersion: monitoring.coreos.com/v1
kind: PodMonitor
metadata:
  name: coroot-node-agent
  namespace: coroot
spec:
  selector:
    matchLabels:
      app.kubernetes.io/name: node-agent
  podMetricsEndpoints:
    - port: http
```

**Comment by emoxam** [comment]
Seems right! Thanks!

**Comment by def** [comment]
This is fixed in the latest helm chart (v0.1.17)
